src/TodoListItem.js

```diff
@@ -1,7 +1,18 @@
 import React from "react";
 
-const TodoListItem = ({ item }) => {
-  return <li>{item.title}</li>;
+const TodoListItem = ({ id, title, onRemoveTodo }) => {
+  const handleRemoveClick = () => {
+    onRemoveTodo(id);
+  };
```

Comment:
This handler function is unnecessary since we're already passing in `onRemoveTodo`